package.json

```diff
@@ -71,8 +71,10 @@
     "backend-windows": "python3 \"events_backend\\manage.py\" runserver",
     "heroku-postbuild": "webpack --config events_frontend/webpack.prod.js",
     "heroku-postbuild-windows": "webpack --config \"events_frontend\\webpack.prod.js\"",
-    "python-migrate": "python3 events_backend/manage.py makemigrations app && python3 events_backend/manage.py migrate",
-    "python-migrate-windows": "python3 \"events_backend\\manage.py\" makemigrations app && python3 \"events_backend\\manage.py\" migrate"
+    "python-migrate": "python events_backend/manage.py makemigrations app && python3 events_backend/manage.py migrate",
+    "python-migrate-windows": "python \"events_backend\\manage.py\" makemigrations app && python3 \"events_backend\\manage.py\" migrate",
+    "python3-migrate": "python3 events_backend/manage.py makemigrations app && python3 events_backend/manage.py migrate",
+    "python3-migrate-windows": "python3 \"events_backend\\manage.py\" makemigrations app && python3 \"events_backend\\manage.py\" migrate"
```

Comment:
We should enforce python3 instead of using whatever is the default on the computer. What's the use case for this?

Reply:
This was specific to my use laptop, where "python" maps to Conda's python version 3.7.3 while "python3" maps to nothing.

Reply:
Then I think it might be more helpful to setup an alias like `alias python3=/path/to/your/conda/python` on your computer rather than adding the risk of not enforcing python3 for the entire repo.